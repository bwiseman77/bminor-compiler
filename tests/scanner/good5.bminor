"~ is okie in side quote"
autointegerarraystring
x++y=2
+2
-2
if()
then
