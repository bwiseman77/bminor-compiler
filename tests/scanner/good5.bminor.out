-------------------------------------------
           token type  text
-------------------------------------------
      STRING_LITERAL  ~ is okie in side quote
               IDENT  autointegerarraystring
               IDENT  x
           INCREMENT  ++
               IDENT  y
              ASSIGN  =
     INTEGER_LITERAL  2
     INTEGER_LITERAL  +2
     INTEGER_LITERAL  -2
                  IF  if
        LEFT_PARENTH  (
       RIGHT_PARENTH  )
               IDENT  then
